week-4/day-1/03supermarketQueue.rb

```diff
@@ -0,0 +1,20 @@
+def queue_time(customers, n)
+  checkout = Array.new(n, 0)
+
+  if customers.empty?
+    return 0
+  else
+    customers.each do |x|
+      checkout[0] = checkout[0] + x
+      checkout.sort!
+    end
+    checkout.last
+  end
+end
+
+puts queue_time([], 1) # 0
```

Comment:
Can you explain me why so many puts?

Reply:
Those are tests. The exercise was available in codewarrior or some webpage like that and they tested the code with these values.